app/styles/layout/l-publications-articles.scss

```diff
@@ -0,0 +1,45 @@
+@import '../settings';
+
+.section-title {
+  margin-top: 0;
+}
+
+.l-publications-news {
+
+  .section-title {
+    margin: 0  0 30px;
+
+    @media #{$mq-tablet} {
+      margin: 0  0 60px;
+    }
+  }
+
+  section {
+    padding: 55px 0 65px;
+    position: relative;
+
+    &::after {
```

Comment:
I use `::after` because lines have to occupy the whole screen width and you can't do that with `border-bottom`. If you have any suggestion about this, it's welcome :)

I don't use `wrap` inside sections, I only use once and its surrounding them :/